pymc_marketing/mmm/mmm.py

```diff
@@ -2409,15 +2381,13 @@ def optimize_budget(
         ----------
         budget : float or int
             The total budget to be allocated.
-        num_periods : float
+        num_periods : int
             The number of time units over which the budget is to be allocated.
-        budget_bounds : dict[str, list[Any]], optional
+        budget_bounds : dict[str, tuple[float, float]], optional
```

Comment:
But this can be `DataArray` from the signature right?

Reply:
Yes, fixed